charts/ccd-data-store-api/values.aat.template.yaml

```diff
@@ -1,10 +1,40 @@
 java:
+  replicas: 2
+  useInterpodAntiAffinity: true
+  image: ${IMAGE_NAME}
+  ingressHost: ${SERVICE_FQDN}
+  keyVaults:
+    ccd:
+      resourceGroup: ccd-shared
+      secrets:
+      - ccd-data-store-api-POSTGRES-PASS
+      - ccd-data-store-api-draftStoreEncryptionSecret
+      - ccd-ELASTIC-SEARCH-URL
+      - ccd-ELASTIC-SEARCH-DATA-NODES-URL
+      - ccd-ELASTIC-SEARCH-PASSWORD
   environment:
-    IDAM_S2S_URL: http://rpe-service-auth-provider-aat.service.core-compute-aat.internal
-    DEFINITION_STORE_HOST: http://ccd-definition-store-api-aat.service.core-compute-aat.internal
-    USER_PROFILE_HOST: http://ccd-user-profile-api-aat.service.core-compute-aat.internal
-    CCD_DM_DOMAIN: ^https?://(?:api-gateway\\.preprod\\.dm\\.reform\\.hmcts\\.net|dm-store-aat\\.service\\.core-compute-aat\\.internal(?::\\d+)?)
-    #IDAM_USER_URL: https://preprod-idamapi.reform.hmcts.net:3511
+    DATA_STORE_DB_HOST: ccd-data-store-api-postgres-db-aat.postgres.database.azure.com
+    DATA_STORE_DB_USERNAME: ccd@ccd-data-store-api-postgres-db-aat
+    DATA_STORE_DB_NAME: ccd_data_store
+    DATA_STORE_DB_OPTIONS: "?stringtype=unspecified&sslmode=require"
+
+    IDAM_S2S_URL: http://rpe-service-auth-provider-java.software-engineering
+    DEFINITION_STORE_HOST: http://ccd-definition-store-api-java
+    USER_PROFILE_HOST: http://ccd-user-profile-api-java
```

Comment:
As these files are used for staging slot deployment, these URLs might dependant on how we deploy the staging slot in AAT ? Will the staging apps be in same namespace? I thought we would put them in a different namespace like "ccd-staging" ? So, may be using  http://ccd-user-profile-api-java.ccd is not a bad idea?

Reply:
probably easiest to keep them in the same namespace so they use the same identity, I assume its safe to always specify the namespace so will add it in anyway